pkg/app/config.go

```diff
@@ -65,5 +65,12 @@ func validateConfig(c Config) error {
 		}
 		registryName[rc.Name] = true
 	}
+
+	switch role := c.Openshift.SandboxRole; role {
+	case apb.ApbRoleAdmin, apb.ApbRoleEdit:
+	default:
+		return fmt.Errorf("sandbox role is not valid - %v", role)
+	}
+
```

Comment:
Exactly what I was trying to describe on your whiteboard. Glad it translated well :)